tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -513,4 +499,4 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUISharingPublicManagement/publicLinkShareByEmail.feature:99](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicManagement/publicLinkShareByEmail.feature#L99)
 
 ### [Uploading folders does not work in files-drop] (https://github.com/owncloud/web/issues/2443)
--   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:248](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L248)
\ No newline at end of file
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:248](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L248)
```

Comment:
I see nothing different

Reply:
i don't see anything different either. I don't know why it's showing like a new change.

Reply:
That's because of added new line L:503

Reply:
That's good. Some people's IDE automatically puts a new-line at the end of files - I like that.
But other people's IDE does not do it!